### DISCUSSION TITLE
Encoding issue

### DISCUSSION THREAD

**Discussion by ankitsoin** [finos/TimeBase-CE#218] [Q&A]
Hi Team,
I am creating fixed stream using QQL in python where I have multiple columns which are defined as of VARCHAR type ( "direction" 'direction' VARCHAR)
I type cast data correctly in python code to write str data into it and also trimmed. Data is sent to TImeBase successfully.
Further I try to retreive data in java or python code using type.raw = False and defining its object class, its retrievable.

But why it is giving me special characters after every varchar column from timebase?
So lets say my data is abc but now it shows as abc\x00

Can any one please help me knowing what I am doing wrong?
Also symbol column is the only column which is coming fine as its internal.

**Comment by alex-karpovich** [comment]
This is known issue, already fixed in EE version, we can move fix to the CE also

**Comment by ankitsoin** [comment]
Yes please, it would be of great help.
Also, request you to please inform once pushed to CE.
Thanks a lot!

**Reply by alex-karpovich** [reply]
Please use https://github.com/epam/TimeBaseClientPython/releases/tag/6.0.2
@ankitsoin

**Reply by ankitsoin** [reply]
Great, thanks a lot for your quick action.
